distributed/http/scheduler/prometheus/__init__.py

```diff
@@ -9,24 +11,33 @@
 class _PrometheusCollector:
     def __init__(self, dask_server):
         self.server = dask_server
+        self.namespace = dask.config.get("distributed.dashboard.prometheus.namespace")
+        self.subsystem = "scheduler"
 
     def collect(self):
         from prometheus_client.core import CounterMetricFamily, GaugeMetricFamily
+        from prometheus_client.metrics import _build_full_name
```

Comment:
Is there any way this can be accessed via a public API call?

Reply:
Sadly no. I was also concerned about this.

Perhaps vendoring in the method would make sense here, it's pretty small and Apache 2.0 licensed.

Reply:
AFAIK we could just copy the code into somewhere like `distributed.http.utils` and include a comment which links to the original source and Apache license on `prometheus_client`.